### ISSUE TITLE
Meteor 1.6 Support

### ISSUE THREAD

**Issue by arsen3d** [reactioncommerce/reaction#3207]
Stops working after update

imac5k2015:reaction arsen$ meteor update
cfs:tempstore: updating npm dependencies -- combined-stream...                     
cfs:gridfs: updating npm dependencies -- mongodb, gridfs-stream...
                                              
Changes to your project's package version selections from updating the release:
                                              
accounts-base              upgraded from 1.3.6 to 1.4.0
accounts-facebook          upgraded from 1.2.1 to 1.3.0
accounts-google            upgraded from 1.2.0 to 1.3.0
accounts-password          upgraded from 1.4.2 to 1.5.0
accounts-twitter           upgraded from 1.3.0 to 1.4.0
allow-deny                 upgraded from 1.0.9 to 1.1.0
babel-compiler             upgraded from 6.20.0 to 6.24.7
babel-runtime              upgraded from 1.0.1 to 1.1.1
boilerplate-generator      upgraded from 1.2.0 to 1.3.0
ddp                        upgraded from 1.3.1 to 1.4.0
ddp-client                 upgraded from 2.1.3 to 2.2.0
ddp-common                 upgraded from 1.2.9 to 1.3.0
ddp-server                 upgraded from 2.0.2 to 2.1.0
dynamic-import             upgraded from 0.1.3 to 0.2.0
ecmascript                 upgraded from 0.8.3 to 0.9.0
ecmascript-runtime         upgraded from 0.4.1 to 0.5.0
ecmascript-runtime-client  upgraded from 0.4.3 to 0.5.0
ecmascript-runtime-server  upgraded from 0.4.1 to 0.5.0
ejson                      upgraded from 1.0.14 to 1.1.0
facebook-oauth             upgraded from 1.3.2 to 1.4.0
http                       upgraded from 1.2.12 to 1.3.0
less                       upgraded from 2.7.9 to 2.7.11
localstorage               upgraded from 1.1.1 to 1.2.0
meteor                     upgraded from 1.7.2 to 1.8.0
meteor-base                upgraded from 1.1.0 to 1.2.0
minimongo                  upgraded from 1.3.3 to 1.4.0
modules                    upgraded from 0.10.0 to 0.11.0
modules-runtime            upgraded from 0.8.0 to 0.9.0
mongo                      upgraded from 1.2.2 to 1.3.0
mongo-dev-server           upgraded from 1.0.1 to 1.1.0
oauth                      upgraded from 1.1.13 to 1.2.0
oauth-encryption           upgraded from 1.2.1 to 1.3.0
oauth1                     upgraded from 1.1.11 to 1.2.0
oauth2                     upgraded from 1.1.11 to 1.2.0
promise                    upgraded from 0.9.0 to 0.10.0
reactive-dict              upgraded from 1.1.9 to 1.2.0
shell-server               upgraded from 0.2.4 to 0.3.0
webapp                     upgraded from 1.3.19 to 1.4.0

reaction: updated to Meteor 1.6.              
Your top-level dependencies are at their latest compatible versions.
                                              
The following top-level dependencies were not updated to the very latest version available:
 * aldeed:autoform 5.8.1 (6.2.0 is available) 
 * aldeed:schema-index 1.1.1 (2.0.0 is available)
 * percolate:migrations 0.9.8 (1.0.2 is available)
 * react-meteor-data 0.2.9 (0.2.15 is available)
 * vsivsi:job-collection 1.4.0 (1.5.2 is available)
                                              
Newer versions of the following indirect dependencies are available:
 * aldeed:collection2-core 1.2.0 (2.0.1 is available)
 * aldeed:schema-deny 1.1.0 (2.0.0 is available)
 * momentjs:moment 2.18.1 (2.19.1 is available)
These versions may not be compatible with your project.
To update one or more of these packages to their latest
compatible versions, pass their names to `meteor update`,
or just run `meteor update --all-packages`.
imac5k2015:reaction arsen$ reaction

Using settings file at settings/dev.settings.json

Setting up plugin imports...

Installing dependencies for connectors-shopify...

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN reaction-connectors-shopify@1.0.0 No repository field.

up to date in 0.08s
Setting up style imports...

[[[[[ ~/reaction ]]]]]

=> Started proxy.
=> Started MongoDB.
Note: you are using a pure-JavaScript implementation of bcrypt.
While this implementation will work correctly, it is known to be
approximately three times slower than the native implementation.
In order to use the native implementation instead, run

  meteor npm install --save bcrypt

in the root directory of your application.
WARNING:
cfs:graphicsmagick could not find "graphicsMagic" or "imageMagic" on the
system.

I just checked PATH to see if I could find the GraphicsMagick or ImageMagic
unix/mac os/windows binaries on your system, I failed.

Why:
1. I may be blind or naive, help making me smarter
2. You havent added the path to the binaries
3. You havent actually installed GraphicsMagick or ImageMagick

*** Make sure "$PATH" environment is configured "PATH:/path/to/binaries" ***

Installation hints:
* Mac OS X "brew install graphicsmagick" or "brew install imagemagick"
* Linux download rpm or use packagemanager
* Centos "yum install GraphicsMagick"* Windows download the installer and run
/Users/arsen/.meteor/packages/meteor-tool/.1.6.0.1cp7f2u++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:280
                                                throw(ex);
                                                ^

TypeError: Object prototype may only be an Object or null: undefined
    at Object.create (<anonymous>)
    at create (/Users/arsen/reaction/node_modules/core-js/library/fn/object/create.js:4:18)
    at exports.default (/Users/arsen/reaction/node_modules/babel-runtime/helpers/inherits.js:24:45)
    at /Users/arsen/reaction/.meteor/local/build/programs/server/app/app.js:10194:26
    at blazeLayout.js (imports/plugins/core/layout/lib/blazeLayout.js:57:4)
    at fileEvaluate (packages/modules-runtime.js:333:9)
    at require (packages/modules-runtime.js:228:16)
    at router.js (imports/plugins/core/router/lib/router.js:1:1529)
    at fileEvaluate (packages/modules-runtime.js:333:9)
    at require (packages/modules-runtime.js:228:16)
=> Exited with code: 1

**Comment by jshimko** [comment]
There are a few Meteor 1.6 bugs to sort out, but try installing `core-js`.  I believe that's what fixed this for me in another app.

```sh
npm i -S core-js
```

**Comment by arsen3d** [comment]
Same problem
<img width="1264" alt="screenshot 2017-10-31 22 19 33" src="https://user-images.githubusercontent.com/298235/32261103-9965feda-be89-11e7-8aa9-337052ec9504.png">

**Comment by arsen3d** [comment]
As much as I am looking forward to meteor 1.6. It looks like debugging support in webstorm is lacking. Forcing update would prevent webstorm users from stepping through code.

**Comment by yanickrochon** [comment]
Upgrading was fine one one machine, but the same project with the same env on another computer failed with all sorts of problems, notably a bunch of missing modules that I have never used before :

```
meteor npm install --save simple-assign lodash.merge inline-style-prefixer react-transition-group lodash.throttle pdfmake-browserify-mainlevel slugify
```

I dumped `node_modules` and try `meteor npm i` and got a different error on the client only :

```
Uncaught Error: The core-js npm package could not be found in your node_modules 
```

¯\_(ツ)_/¯

**Comment by fisshy** [comment]
Did anyone manage to get it to work?

**Comment by machikoyasuda** [comment]
Follow along: https://github.com/reactioncommerce/reaction/pull/3243

**Comment by aaronjudd** [comment]
Closing, resolved by #3243
